src/schema/schema_create.c

```diff
@@ -1098,9 +1126,13 @@ __schema_create(WT_SESSION_IMPL *session, const char *uri, const char *config)
      */
     WT_RET(__wt_meta_track_on(session));
     if (import) {
-        F_SET(session, WT_SESSION_IMPORT);
+        if (!F_ISSET(session, WT_SESSION_IMPORT)) {
+            F_SET(session, WT_SESSION_IMPORT);
+            import_flag_set = true;
```

Comment:
Minor: I suggest 1. moving the comment about the recursive call up to here and 2. naming the boolean `clear_import_flag`.

Reply:
Agree. Done.